src/index.js

```diff
@@ -168,19 +201,20 @@ class ExtractTextPlugin {
                 extractedChunk.addGroup(chunkGroup);
               }
 
```

Comment:
We add the ChunkGroups to the extracted chunk and the _children and _parents sets belong to ChunkGroup now. Please confirm that these lines are unnecessary.